### PR TITLE
workflow: Delete assign_project.yml

### PR DESCRIPTION
**Pull request by tisonkun**
I'm not quite sure but this action consumer a lot of our quota on GitHub API queries.

I don't see anyone looking into these projects any more, please confirm and if so, remove the action.

cc @bb7133 @qw4990 @XuHuaiyu @jebter @zhouqiang-cl 

### Release note

```release-note
None
```

### CODE REVIEW

File: .github/workflows/assign_project.yml [hunk 1]
```diff
@@ -1,63 +0,0 @@
...
-    types: [labeled]
-env:
-  GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-
-jobs:
-  assign_one_project:
-    runs-on: ubuntu-latest
-    name: Assign to One Project
-    steps:
-    - name: Run issues assignment to project SIG Runtime Kanban
```
Comment (XuHuaiyu) [inline .github/workflows/assign_project.yml:14]: LGTM

**Comment by tisonkun** [comment]
@XuHuaiyu thanks for your review. Could you approve explicitly by GitHub mechanism?

**Review by XuHuaiyu — APPROVED** [review]
LGTM

File: .github/workflows/assign_project.yml [hunk 2]
```diff
@@ -1,63 +0,0 @@
...
-      uses: srggrs/assign-one-project-github-action@1.2.0
-      if: |
-        contains(github.event.issue.labels.*.name, 'component/coprocessor') ||
-        contains(github.event.issue.labels.*.name, 'sig/executor') ||
-        contains(github.event.issue.labels.*.name, 'component/expression')
-      with:
-        project: 'https://github.com/pingcap/tidb/projects/38'
-        column_name: 'Issue Backlog: Need Triage'
-    - name: Run issues assignment to project SIG Planner Kanban
```
Comment (qw4990) [inline .github/workflows/assign_project.yml:23]: LGTM

**Review by qw4990 — APPROVED** [review]

**Comment by tisonkun** [comment]
Thanks for your confirming!

@bb7133 @zhouqiang-cl could you help on merging this PR？

**Pull request merged**